packages/office-ui-fabric-react/.storybook/main.js

```diff
@@ -5,4 +5,5 @@ module.exports = {
     return custom({ config });
   },
   addons: ['@storybook/addon-a11y/register'],
+  addons: ['storybook-addon-performance/register'],
```

Comment:
These should be combined. Otherwise you're overwriting the first entry with the second one.
```suggestion
  addons: ['@storybook/addon-a11y/register', 'storybook-addon-performance/register'],
```

Reply:
(I just fixed it by applying the suggestion)